apps/evans/components/core/AudioPlayer.tsx

```diff
@@ -42,93 +37,101 @@ const AudioPlayer: React.FC<Props> = ({ tracks }) => {
-              />
-            </button>
-            <div className="flex-grow bg-flprimary/5 py-2 px-6 rounded-full flex justify-start overflow-hidden relative mx-8 hidden md:block md:order-1">
-              <span className="text-xl font-bold text-flprimary whitespace-nowrap">
-                {currentTitle}
-              </span>
-              <div className="absolute h-full w-24 right-0 top-0 bg-gradient-to-l from-[#F7F5F5] to-transparent" />
-            </div>
-            <button
-              onClick={() => {
-                setCurrentTime(currentTime - 10);
-                audio.currentTime = currentTime - 10;
+      <audio ref={audioRef} src={track.mp3Url} />
+      <div
+        className={cx(
+          `flex flex-col gap-4`,
+          tracks.length > 1 && `mb-8`,
+          !audio && `opacity-75 cursor-not-allowed [&>*]:pointer-events-none`,
+        )}
+      >
+        <div className="flex items-center justify-center md:justify-between">
+          <button
+            onClick={() => setIsPlaying(!isPlaying)}
+            className="w-12 h-12 bg-flprimary rounded-full flex justify-center items-center *hover:opacity-90 transition-[opacity,transform] duration-200 active:scale-95 active:!opacity-100 shrink-0 order-1 md:order-0 mx-8 md:mx-0"
+          >
+            <i
+              aria-hidden
+              className={cx(
+                `fa-solid text-xl text-white pt-px`,
+                isPlaying ? `fa-pause` : `fa-play pl-[3px]`,
+              )}
+            />
+          </button>
+          <div className="flex-grow bg-flprimary/5 py-2 px-5 rounded-full flex justify-start overflow-hidden relative mx-6 hidden md:block md:order-1">
+            <span className="text-xl font-bold text-flprimary whitespace-nowrap">
+              {track.title}
+            </span>
+            <div className="absolute h-full w-24 right-0 top-0 bg-gradient-to-l from-[#F7F5F5] to-transparent" />
+          </div>
+          <button
+            onClick={() => {
+              setCurrentTime(currentTime - 15);
+              audio && (audio.currentTime = currentTime - 15);
+            }}
+            className="w-10 h-10 rounded-full hover:bg-flprimary/10 active:bg-flprimary/20 active:scale-95 transition-[background-color,transform] flex justify-center items-center md:mr-2 order-0 md:order-2 shrink-0"
+          >
+            <i aria-hidden className="w-6 h-6 text-lg text-flprimary fa fa-rotate-left" />
+          </button>
+          <button
+            onClick={() => {
+              setCurrentTime(currentTime + 15);
+              audio && (audio.currentTime = currentTime + 15);
+            }}
+            className="w-10 h-10 rounded-full hover:bg-flprimary/10 active:bg-flprimary/20 active:scale-95 transition-[background-color,transform] flex justify-center items-center order-2 md:order-3 shrink-0"
+          >
+            <i
+              aria-hidden
+              className="w-6 h-6 text-lg text-flprimary fa fa-rotate-right"
```

Comment:
i changed your icon here, for two reasons.  1, i was a little confused by it, i thought it would advance me to the next _track_. i think these are more clear that they skip forward/back a bit. and 2) because they match the native app controls, which i think is sortof nice to slightly strive for consistency between the two audio UIs.